ts/src/bitmex.ts

```diff
@@ -624,10 +629,12 @@ export default class bitmex extends Exchange {
                     'swap': swap,
                     'future': future,
                     'option': false,
+                    'index': index,
                     'active': active,
                     'contract': contract,
-                    'linear': contract ? !inverse : undefined,
-                    'inverse': contract ? inverse : undefined,
+                    'linear': linear,
+                    'inverse': inverse,
+                    'quanto': isQuanto,
```

Comment:
this seems to be needed, as 'isQuanto' value is provided by exchange itself.
but I am now verifying whether isQuanto surely excludes linear markets

Reply:
i.e. `DOT/USD:BTC` contract market or dozens other markets are quanto